### PR TITLE
Update 2018-10-30-why-you-need-to-implement-a-plugin.md

### PR DESCRIPTION
**Pull request by LouisLouw1**
All right, I hope I'm doing this correctly!

### CODE REVIEW

File: _posts/2018-10-30-why-you-need-to-implement-a-plugin.md [hunk 1]
```diff
@@ -4,46 +4,46 @@ title:  "Why you need to implement your own IntelliJ IDEA plugin"
...
-IntelliJ IDEA is a powerful tool that aims to increase
-productivity. It speeds up development process and optimizes 
-most of the standard use cases like code formatting, code generation, 
-navigation, etc. If one wants to go further they might tweak some 
-other areas. For instance, I set up my IDE to generate tests prefixed with 
-word 'should'. It also automatically [adds annotations to test classes 
+IntelliJ IDEA is a powerful tool that aims at increasing
+productivity. It speeds up the development process and optimizes 
+most of the standard use cases such as code formatting, code generation, 
+navigation, etc. However, the utility of the IntelliJ IDEA is not limited
```
[the remark this answers is not in the document]
Comment (LouisLouw1) [reply _posts/2018-10-30-why-you-need-to-implement-a-plugin.md]: Alright, I can change it to their site
```suggestion
navigation, etc. However, their site is not limited 
```

File: _posts/2018-10-30-why-you-need-to-implement-a-plugin.md [hunk 2]
```diff
@@ -4,46 +4,46 @@ title:  "Why you need to implement your own IntelliJ IDEA plugin"
...
-most of the standard use cases like code formatting, code generation, 
-navigation, etc. If one wants to go further they might tweak some 
-other areas. For instance, I set up my IDE to generate tests prefixed with 
-word 'should'. It also automatically [adds annotations to test classes 
+IntelliJ IDEA is a powerful tool that aims at increasing
+productivity. It speeds up the development process and optimizes 
+most of the standard use cases such as code formatting, code generation, 
+navigation, etc. However, the utility of the IntelliJ IDEA is not limited 
+to these processes. For instance, I set up my IDE to generate tests prefixed with 
+the word 'should'. The plugin also automatically [adds annotations to test classes
```
[the remark this answers is not in the document]
Comment (LouisLouw1) [reply _posts/2018-10-30-why-you-need-to-implement-a-plugin.md]:
```suggestion
the word 'should'. The IDE (IntelliJ IDEA) tool also automatically [adds annotations to test classes 
```

File: _posts/2018-10-30-why-you-need-to-implement-a-plugin.md [hunk 3]
```diff
@@ -4,46 +4,46 @@ title:  "Why you need to implement your own IntelliJ IDEA plugin"
...
 %}
 
-If you work within a team for some time you for sure have some
-coding standards, architectural standards, some other patterns.
-If it is a web application with Spring you might 
-break your code into repository, service and controller levels.
-Or let's say you have Event Driven architecture then you might have
-some standard event classes, aggregate classes, etc. 
-In all of these cases you can always find a pattern which you can automate. 
+If you work within a team for a certain amount of time, you are bound
```
[the remark this answers is not in the document]
Comment (LouisLouw1) [reply _posts/2018-10-30-why-you-need-to-implement-a-plugin.md]: I can change the phrasing, sure. How about: A team that works together continuously naturally develops ...
```suggestion
A team that works continuously together naturally develops 
```

File: _posts/2018-10-30-why-you-need-to-implement-a-plugin.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Shpota) [reply _posts/2018-10-30-why-you-need-to-implement-a-plugin.md]: You can change it either to "it" or to "IntelliJ IDEA". I was talking about using definitive article "the". In the official site they call their product "IntelliJ IDEA", not "the IntelliJ IDEA". That was the point.

File: _posts/2018-10-30-why-you-need-to-implement-a-plugin.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Shpota) [reply _posts/2018-10-30-why-you-need-to-implement-a-plugin.md]: Let's leave either "The IDE" or "IntelliJ IDEA" or "It". Clarification in brackets is not needed here, it is clear form the context.

**Comment by LouisLouw1** [comment]
In the meanwhile, I understand the "un/resolving conversation" part as well. The accidently resolved the conversations without actually having changed a thing to the post. The commits above now have the changes you requested.

**Pull request merged**